src/assets/scripts/client/navigationLibrary/NavigationLibrary.js

```diff
@@ -163,6 +203,111 @@ class NavigationLibrary {
         );
     }
 
+    /**
+     * Generate lines for SIDs and add them to the procedure lines
+     *
+     * @for NavigationLibrary
+     * @method _initializeSidLines
+     */
+    _initializeSidLines() {
+        const sids = this.getProceduresByType(PROCEDURE_TYPE.SID);
+        const sidLines = [];
+        let mostRecentFixName = '';
+
+        // TODO: simplify/rector these nested loops.
+        for (let i = 0; i < sids.length; i++) {
+            const sid = sids[i];
+            const lines = [];
+            const exits = [];
+
+            for (let j = 0; j < sid.draw.length; j++) {
+                const fixList = sid.draw[j];
+                const positions = [];
+
+                for (let k = 0; k < fixList.length; k++) {
+                    const fixName = fixList[k];
+                    mostRecentFixName = fixName;
+
+                    if (fixName.indexOf('*') !== INVALID_INDEX) {
+                        mostRecentFixName = fixName.replace('*', '');
+                        exits.push(mostRecentFixName);
+                    }
+
+                    const fixPosition = this.getFixRelativePosition(mostRecentFixName);
+
+                    if (!fixPosition) {
+                        console.warn(`Unable to draw line to '${fixName}' because its position is not defined!`);
+                        continue;
+                    }
```

Comment:
Excellent, _thank you_ for this breadcrumb. I guarantee it will be printed occasionally and people will appreciate it :smiley: